src/operations.rs

```diff
@@ -46,6 +52,35 @@ pub(crate) fn database_create(context: &mut ConsoleContext, input: &[String]) ->
     Ok(())
 }
 
+pub(crate) fn database_import(context: &mut ConsoleContext, input: &[String]) -> CommandResult {
+    let driver = context.driver.clone();
+    let db_name = input[0].clone();
+    let schema = read_to_string(input[1].clone()).map_err(|err| Box::new(err) as Box<dyn Error + Send>)?;
+    let data_file_path: PathBuf = input[2].clone().into();
```

Comment:
I would use `context.read_path` or whatever it's called, to relativize the path correctly (for example, if this is running inside a script, it will relativize relative to the script location).

Basically doing that for all paths now